app/Keycloak/Service/ApiClient.php

```diff
@@ -11,7 +11,7 @@
 
 interface ApiClient
```

Comment:
Should this interface move to the `Client` namespace instead of sitting inside the `Service` namespace?

Reply:
Also the concrete implementation `KeycloakApiClient` can move.